website/docs/Pipelines/Imputation_Pipeline/README.md

```diff
@@ -0,0 +1,133 @@
+---
+sidebar_position: 1
+---
+ 
+# Imputation Overview
+ 
+| Pipeline Version | Date Updated | Documentation Author | Questions or Feedback |
+| :----: | :---: | :----: | :--------------: |
+| [Imputation_v1.0.0](https://github.com/broadinstitute/warp/releases) | August, 2021 | [Elizabeth Kiernan](mailto:ekiernan@broadinstitute.org) | Please file GitHub issues in warp or contact [Kylee Degatano](mailto:kdegatano@broadinstitute.org) |
+ 
+## Introduction to the Imputation pipeline
+The Imputation pipeline imputes missing genotypes from a multi-sample VCF using a large genomic reference panel. It is based on the Michigan Imputation Server pipeline. Overall, the pipeline filters, phases, and performs imputation on a multi-sample VCF. It outputs the imputed VCF along with key imputation metrics.
```

Comment:
Do we want to mention that this can also take in an array of single sample VCFs here? @ekiernan

Reply:
Yes, updated.